src/test/java/testInhertiance/testBusiness.java

```diff
@@ -0,0 +1,125 @@
+package testInhertiance;
+
+import javaFundaments.inhertiance.*;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class testBusiness {
+
+    @Test
+    public void testNewRestaurant() {
+        Restaurant restaurant = new Restaurant("Restaurant", "Here", 5, "Food");
+        List<Review> actual = restaurant.getReviews();
+        List<Review> expected = new ArrayList<>();
+
+        assertEquals("New Restaurant did not have an empty review list", expected, actual);
+    }
+
+    @Test
+    public void testNewShop() {
+        Shop shop = new Shop("Shop", "Here", 5, "Stuff");

```

Comment:
This new shop should have 3+ assert statements. Next time make 2 more new shops!